WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -186,8 +186,9 @@ private extension PersonViewController {
     }
 
     func removeWasPressed() {
+        let title = NSLocalizedString("Remove", comment: "Remove Alert Title") + " " + person.username
```

Comment:
I think you might want to do this as a formatted string – `"Remote %@"`. It's possible that in other languages the order of the words might be switched around?